openlibrary/core/vendors.py

```diff
@@ -43,92 +44,85 @@ def search_amazon(title='', author=''):
 
     results = lending.amazon_api.search(Title=title, Author=author, SearchIndex='Books')
     data = {'results': []}
-    try:
-        for product in results:
-            data['results'].append(_serialize_amazon_product(product))
-    except SearchException:
-        data = {'error': 'no results'}
+    #try:
+    #    for product in results:
+    #        data['results'].append(_serialize_amazon_product(product))
+    #except SearchException:
+    #    data = {'error': 'no results'}
     return data
 
-
 def _serialize_amazon_product(product):
     """Takes a full Amazon product Advertising API returned AmazonProduct
     with multiple ResponseGroups, and extracts the data we are interested in.
 
     :param amazon.api.AmazonProduct product:
     :return: Amazon metadata for one product
     :rtype: dict
-    """
-
-    price_fmt = price = qlt = None
-    used = product._safe_get_element_text('OfferSummary.LowestUsedPrice.Amount')
-    new = product._safe_get_element_text('OfferSummary.LowestNewPrice.Amount')
 
-    # prioritize lower prices and newer, all things being equal
-    if used and new:
-        price, qlt = (used, 'used') if int(used) < int(new) else (new, 'new')
-    # accept whichever is available
-    elif used or new:
-        price, qlt = (used, 'used') if used else (new, 'new')
+    {
+      'price': '$54.06',
+      'price_amt': 5406,
+      'physical_format': 'Hardcover',
+      'authors': [{'role': 'Author', 'name': 'Guterson, David'}],
```

Comment:
Unfortunately I don't think this `role` is going to do anything here. It's good that it's available from Amazon, but #3084 needs to be addressed to make use of it. Probably not something you want to do jsut yet.